.travis.yml

```diff
@@ -33,7 +33,7 @@ env:
     - DRIVER=hiredis   REDIS_BRANCH=3.2
     - DRIVER=synchrony REDIS_BRANCH=3.0
     - DRIVER=synchrony REDIS_BRANCH=3.2
-    - DRIVER=ruby      REDIS_BRANCH=unstable
+    - DRIVER=ruby      REDIS_BRANCH=stable
```

Comment:
In think the intent of building `unstable` was to test `4.0`. The build matrix could simply be `3.0 3.2 4.0`.